Linphone failure: There is no object pool created in thread
If I let my application sit for a while or after a couple calls I get the following crash on linphone: 2016-11-09 09:21:17:372 ortp-message-Listening point [0x1740fe480] on [sip:0.0.0.0:5060;transport=UDP] destroyed 2016-11-09 09:21:17:372 ortp-message-Listening point [0x1740fe500] on [sip:0.0.0.0:5060;transport=TCP] destroyed 2016-11-09 09:21:17:373 ortp-message-Listening point [0x1740fe580] on [sip:0.0.0.0:49391;transport=TLS] destroyed 2016-11-09 09:21:17:373 ortp-message-sal_unlisten_ports done 2016-11-09 09:21:17:373 ortp-message-http provider destroyed. 2016-11-09 09:21:17:373 ortp-message-stack [0x17434dd60] destroyed. 2016-11-09 09:32:44.566120 Setec[249:9352] [MC] Invalidating cache 2016-11-09 09:32:44.567110 Setec[249:7759] [MC] Reading from public effective user settings. SetupCalls 2016-11-09 09:45:36.743273 Setec[249:10714] MakeCall.. 2016-11-09 09:45:36.743842 Setec[249:10714] identity: sip:5612954779@phoneserverhere 2016-11-09 09:45:36:744 ortp-warning-There is no object pool created in thread [6140047360]. Use belle_sip_object_pool_push() to create one. Unowned objects not unref'd will be leaked. The code I am using is:  DispatchQueue.global(qos: .background).async { NSLog("MakeCall..") guard let _ = self.setIdentify() else { print("no identity in makeCall") return } linphone_core_invite(lc, numberToCall) self.mainLoop(sec: 5) self.shutdown() } and it gets caught in the guard.